src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -34,7 +34,7 @@ public DBFTPlugin()
 
         public DBFTPlugin(Settings settings) : this()
         {
-            this.settings = settings;
+            this._settings = settings;
```

Comment:
You should remove `this`. Dotnet coding standards, 99% of professional c# developers dont use that method anymore.